docs/guide/using_existing_contracts.rst

```diff
@@ -18,6 +21,19 @@ This is how we can interact with it:
     :language: python
     :dedent: 4
 
+
+Raw contract calls
+------------------
+
+If you do not have ABI statically, but you know the interface of the contract on StarkNet, you can make a raw call:
```

Comment:
Let's extend this description to indicate that it's impossible to use `Contract` api without an abi (contract functions won't generate), and that's why client has to be used directly.